stor/setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import setup
```

Comment:
I think this one may need to have an `install_requires==` to specific versions of required sub-packages